src/Objects/Match.ts

```diff
@@ -372,14 +384,25 @@ export class Match {
         if (this.competition)
             lines.push(`Event: ${this.competition.getName()}`);
 
-        lines.push("");
+        // Add match link if HTML is enabled
```

Comment:
What's the advantage of this change? I don't see the added value, and it's less readable to me.